### PR TITLE
Niloofar / moving besquare page under deriv careers

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by sean-binary** [comment]
Hello @niloo-fs, there should also be navigations to that page on the footer and off-canvas menu component 😄

**Comment by niloo-fs** [comment]
> Hello @niloo-fs, there should also be navigations to that page on the footer and off-canvas menu component smile



> Hello @niloo-fs, there should also be navigations to that page on the footer and off-canvas menu component smile

Hello @sean-binary, I can not see those link on the footer and off-canvas menu component. if there is a navigation there would u pls share a picture of it.

**Comment by sean-binary** [comment]
@niloo-fs My bad, you are right. For Besquare page itself, we didn't have those navigations globally, just inside the header of career's page. You can ignore the previous comment 👍

File: src/pages/careers/besquare/components/helper/_card.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const Card = ({ card_content, custom_icon, has_list, style, title_component }) =
     return (
         <CardWrapper {...card_wrapper}>
             <IconWrapper {...icon_wrapper} src={custom_icon?.src || card_content.src} alt="" />
-            {title_component && title_component}
```
Comment (habib-deriv) [inline src/pages/careers/besquare/components/helper/_card.js:53]: Hi @niloo-fs, Any reason, you removed the availability check of this title_component ?

Comment (habib-deriv) [reply src/pages/careers/besquare/components/helper/_card.js:53]: All right! Looks like the check was unnecessary.

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**